src/Redux/Selectors/MountedComponentsSelectors.js

```diff
@@ -0,0 +1 @@
+export const isMountedComponentSelector = name => state => state.MountedComponents[name];
```

Comment:
about the name `isMountedComponentSelector` - 
if it's "MountedComponent" because it is meant to **conform to the name of `MountedComponents` slice**, maybe add an 's' at the end of "MountedComponent"?
if it's meant to be **singular** to enhance readability, so that we'll know it's referring to a mounting state of a **single** component - maybe change it to "isComponentMountedSelector"?

Reply:
it stays for one component. i will change it as you wish.